prow.sh

```diff
@@ -872,10 +872,17 @@ install_snapshot_controller() {
   cnt=0
   expected_running_pods=$(kubectl apply --dry-run=client -o "jsonpath={.spec.replicas}" -f "$SNAPSHOT_CONTROLLER_YAML")
   expected_namespace=$(kubectl apply --dry-run=client -o "jsonpath={.metadata.namespace}" -f "$SNAPSHOT_CONTROLLER_YAML")
-  while [ "$(kubectl get pods -n "$expected_namespace" -l app=snapshot-controller | grep 'Running' -c)" -lt "$expected_running_pods" ]; do
+  expect_key='app'
+  expected_label=$(kubectl apply --dry-run=client -o "jsonpath={.spec.template.metadata.labels['$expect_key']}" -f "$SNAPSHOT_CONTROLLER_YAML")
+  if [ -z $expect_label ]; then
+    expect_key='app\.kubernetes\.io/name'
```

Comment:
![image](https://github.com/kubernetes-csi/csi-release-tools/assets/9926112/2a46dc8e-ed0f-4009-8723-4a7d65d95250)

I tried it locally, and I found that I still need to add it, if not, it will not get the value

Reply:
Interesting, I never noticed that. Thanks for checking!